src/pages/state.tsx

```diff
@@ -238,7 +240,7 @@ const State: React.FC = () => {
             }))}
             limit={pageLimit}
             offset={pageOffset}
-            paginationEnable={true}
+            paginationEnable={paginationCount >= 5}
```

Comment:
@rahulg1254 add this default size in app.constant

Reply:
Resolved